common/src/main/java/org/apache/celeborn/common/util/PbSerDeUtils.java

```diff
@@ -100,8 +107,18 @@ public static ConcurrentHashMap<String, FileInfo> fromPbFileInfoMap(byte[] data)
       throws InvalidProtocolBufferException {
     PbFileInfoMap pbFileInfoMap = PbFileInfoMap.parseFrom(data);
     ConcurrentHashMap<String, FileInfo> fileInfoMap = new ConcurrentHashMap<>();
+    ConcurrentHashMap<String, UserIdentifier> cache = new ConcurrentHashMap<>();
```

Comment:
data here contains fileInfos of one shuffleKey, which means they belong to one user, so I think it's unnecessary to use a map here. If we want, we should create the map in reloadAndCleanFileInfos and pass the cache map to each call to fromPbFileInfoMap.